bedrock/firefox/templates/firefox/releases/notes.html

```diff
@@ -210,6 +210,19 @@ <h3>“Firefox isn’t just fast and smooth, it also has lots of features and op
       </div>
     {% endif %}
 
+    {% if variation == 'c' %}
+      <div id="social-proof" class="mzp-l-content mzp-has-sidebar mzp-l-sidebar-left">
+        <div class="mzp-l-main mzp-l-article">
+          {% call call_out_compact(
+            title=download_title,
+            class=product_class
+          ) %}
+            {{ download_button }}
+          {% endcall %}
+        </div>
+      </div>
+    {% endif %}
```

Comment:
<img width="966" alt="Screen Shot 2020-04-23 at 2 24 16 PM" src="https://user-images.githubusercontent.com/30483988/80151248-958a7f80-856e-11ea-9f8d-952835721228.png">

Looks a bit squashed at certain view widths. Suggest making this full-width with no sidebar.

```suggestion
    {% if variation == 'c' %}
      <div id="social-proof">
        {% call call_out_compact(
          title=download_title,
          class=product_class
        ) %}
          {{ download_button }}
        {% endcall %}
      </div>
    {% endif %}
```

<img width="1341" alt="Screen Shot 2020-04-23 at 2 29 19 PM" src="https://user-images.githubusercontent.com/30483988/80151504-06ca3280-856f-11ea-9162-5b65f49e8a1a.png">